app/tab-tracking.R

```diff
@@ -2,21 +2,19 @@ source("ui_components.R")
 
 tab_tracking <- tabItem(
   tabName = "tracking",
-  fluidRow(h2("Cromwell Workflow Tracking"), align = "center"),
   fluidRow(
```

Comment:
All of these 3 inputs being in the same row means that it's a little weird clicking up and down arrows on the far right of the screen with the number being way over to the left of the screen. I'd revert this change

Reply:
Ah, on my they were stacked very nicely.  Happy to revert b/c that sounds dumb looking.

Reply:
on second thought we can set the width of the text box, so that might be better